www/src/components/CodeEditor/CodeEditor.module.scss

```diff
@@ -75,8 +95,33 @@
   background-color: $c-divider;
 }
 
+.PreviewHeader {
+  @include f-note;
+  padding-top: 10px;
+  padding-bottom: 5px;
```

Comment:
Maybe you can explain to me why we're doing this sort of thing with the vertical alignment. It feels brittle and non-intuitive.

For instance, `padding: 10px` would be nice but instead we have to do `padding: 10px 10px 5px` because, for some reason, we've set:
```css
:before {
    margin-top: -2px;
}
:after {
    margin-top: 2px;
}
```
Why do we add this layer of complication? It looks _slightly_ different but to my eye, no better or worse- and it makes paddings/margins more complicated to manage.